### ISSUE TITLE
Question where is the finetune model checkpoint located?

### ISSUE THREAD

**Issue by GUUser91** [SWivid/F5-TTS#83]
I followed this user's instructions for preparing a dataset
https://github.com/SWivid/F5-TTS/issues/62#issuecomment-2411546271
Then I lowered the batch size to 700 and learning rate to 1e-4
Then I started the training with this command
```
python train.py
```
I finished the training but I can't find the model.

**Comment by lpscr** [comment]
@GUUser91 the model save in **F5-TTS** in **ckpts/E2TTS_Base** or **ckpts/F5TTS_Base**

if you can't find the problem it's i guest you have very big number in checkpoint per steps 

change this number in train.py 

```
save_per_updates = 50000  # save checkpoint per steps
last_per_steps = 5000  # save last checkpoint per steps
```

If the save_stage was not reached, the last checkpoint will not be saved, which is why you are not getting any files.

see here 
in model/trainer.py 

https://github.com/SWivid/F5-TTS/blob/0297be2541f9a062f9d54103926bdb88d63440ea/model/trainer.py#L244-L250

**Comment by lpscr** [comment]
When the training is finished, if there isn't any saved checkpoint,  must be at least one **last_model.pt** file 
this need update .

**Comment by GUUser91** [comment]
@lpscr
Thanks.
